packages/fxa-payments-server/src/lib/apiClient.ts

```diff
@@ -22,6 +22,7 @@ export interface MetricsContext {
   utmCampaign?: string;
   utmContext?: string;
   utmMedium?: string;
+  utmReferrer?: string;
```

Comment:
Is this used somewhere? I think this is the only reference to `utmReferrer` in this PR.